### DISCUSSION TITLE
Is there a way to toggle between Telegram and Whatsapp clients?

### DISCUSSION THREAD

**Discussion by ryamgoh** [d99kris/nchat#269] [Q&A]
I am a user of both messaging apps and I couldn't find a way to toggle between the 2 clients without removing and setting up (not a fan of having both WhatsApp and Telegram on the same interface)

Thank you for the help!

**Comment by d99kris** [comment]
Hi @ryamgoh - yes this is possible, and it's how I primarily use nchat myself. The way to do this with nchat is to have one config directory per protocol (and/or phone number). To simplify usage one can set up aliases (in `~/.bashrc` or other file depending on shell used), for example:
```
alias telegram='nchat -d ~/.nchat-telegram'
alias whatsapp='nchat -d ~/.nchat-whatsapp'
```
Then use regular set up for them (separately), for example:
```
telegram -s
whatsapp -s
```

I will look into documenting the above in the README at some point..

**Reply by ryamgoh** [reply]
Thanks a lot, was looking for this answer. Indeed, the README can be clearer! Have a nice day :clinking_glasses:
